docs/application/vstools-mac/README.md

```diff
@@ -0,0 +1,147 @@
+# Installing Visual Studio for Mac Extension for Tizen
```

Comment:
It is minor...
Recently, we use a **verb** for section titles.

Reply:
Done